components/financial_assistance/app/models/financial_assistance/subscribers/application_eligibility_response.rb

```diff
@@ -14,7 +14,19 @@ def self.subscription_details
       def call(_event_name, _e_start, _e_end, _msg_id, payload)
         stringed_key_payload = payload.stringify_keys
         xml = stringed_key_payload['body']
-        application = FinancialAssistance::Application.where(:hbx_id => stringed_key_payload['assistance_application_id']).first if stringed_key_payload['assistance_application_id'].present?
+
+        if stringed_key_payload['assistance_application_id'].present?
+          applications = FinancialAssistance::Application.where(:hbx_id => stringed_key_payload['assistance_application_id'])
+          if applications.count == 1
```

Comment:
https://github.com/ideacrew/enroll/pull/277 Can you change this to `.any?` We tried factoring out `.count` a little while back (see this PR).

Reply:
@ZASMan Here, in this context we only want to move forward if there is only one application with matching hbx_id and if there are more or less then we do not want to move forward. This is the reason why I am checking the count here.